modules/user_data/templates/01_disk_management.sh.tpl

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
```

Comment:
Some documentation would be useful here imo, those scripts contain quite a complicated logics and if someone from the outside reads them it would be very confusing. Just a summary would be enough maybe for each template?

Reply:
sure I'll add some